dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/internal/data/TraceWriterTest.kt

```diff
@@ -86,144 +80,23 @@ internal class TraceWriterTest {
-                    spansList[index].operationName,
-                    spansList[index].tags[DDTags.ERROR_TYPE].toString(),
-                    spansList[index].tags[DDTags.ERROR_MSG].toString()
-                )
-            )
-        }
-        attributesCaptor.allValues.forEachIndexed { index, map ->
-            assertThat(map).containsEntry(
-                RumAttributes.INTERNAL_ERROR_TYPE,
-                spansList[index].tags[DDTags.ERROR_TYPE].toString()
-            )
-        }
-        stackTracesCaptor.allValues.forEachIndexed { index, stacktrace ->
-            assertThat(stacktrace).isEqualTo(spansList[index].tags[DDTags.ERROR_STACK].toString())
-        }
-        spansList.forEach {
-            it.finish()
-        }
-    }
-
-    @Test
-    fun `M send a RUM Error event W onWritingErrorSpan {no error type}(`(forge: Forge) {
-        // GIVEN
-        val spansList = ArrayList<DDSpan>(2).apply {
-            add(
-                forgeErrorSpan(forge).apply {
-                    this.context().setTag(DDTags.ERROR_TYPE, null)
-                }
-            )
-            add(
-                forgeErrorSpan(forge).apply {
-                    this.context().setTag(DDTags.ERROR_TYPE, null)
-                }
-            )
-        }
-
-        // WHEN
-        testedWriter.write(spansList)
-
-        // THEN
-        val errorMessagesCaptor = argumentCaptor<String>()
-        val stackTracesCaptor = argumentCaptor<String>()
-        verify(mockAdvancedRumMonitor, times(2)).addErrorWithStacktrace(
-            errorMessagesCaptor.capture(),
-            eq(RumErrorSource.SOURCE),
-            stackTracesCaptor.capture(),
-            eq(emptyMap())
-        )
-        errorMessagesCaptor.allValues.forEachIndexed { index, message ->
-            assertThat(message).isEqualTo(
-                TraceWriter.SPAN_ERROR_WITH_MESSAGE_FORMAT.format(
-                    Locale.US,
-                    spansList[index].operationName,
-                    spansList[index].tags[DDTags.ERROR_MSG].toString()
-                )
-            )
-        }
-
-        stackTracesCaptor.allValues.forEachIndexed { index, stacktrace ->
-            assertThat(stacktrace).isEqualTo(spansList[index].tags[DDTags.ERROR_STACK].toString())
-        }
-        spansList.forEach {
-            it.finish()
-        }
-    }
-
-    @Test
-    fun `M send a RUM Error event W onWritingErrorSpan {no error message}(`(forge: Forge) {
-        // GIVEN
-        val spansList = ArrayList<DDSpan>(2).apply {
-            add(
-                forgeErrorSpan(forge).apply {
-                    this.context().setTag(DDTags.ERROR_MSG, null)
-                }
-            )
-            add(
-                forgeErrorSpan(forge).apply {
-                    this.context().setTag(DDTags.ERROR_MSG, null)
-                }
-            )
+            forgeErrorSpan(forge).apply {
+                this.context().setTag(DDTags.ERROR_TYPE, null)
```

Comment:
why do you set these to null ?

Reply:
To test different types of errors (like there was in the legacy tests